test/facets/Token/behaviors/batchDisableAddressLength.t.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.19;
+
+import { ITokenBridgeInternal } from "../../../../contracts/facets/Token/ITokenBridgeInternal.sol";
+import { ITokenBridge } from "../../../../contracts/facets/Token/ITokenBridge.sol";
+import { ArbForkTest } from "../../../ArbForkTest.t.sol";
+import { TokenBridge } from "../TokenBridge.t.sol";
+import { IOwnableInternal } from "@solidstate/contracts/access/ownable/IOwnableInternal.sol";
+
+/// @title Token_batchDisableAddressLength
+/// @notice it tests the batchDisableAddressLength function of the TokenBridge
+contract Token_batchDisableAddressLength is ArbForkTest, TokenBridge {
+    event AddressLengthsDisabled(uint256 indexed mask);
+
+    address public ALICE = makeAddr("Alice");
+    address public OWNER = makeAddr("Owner");
+    address public tokenAddress;
+    uint256 public constant EVM_ADDRESS_LENGTH = 42;
+    uint256 public constant SOLANA_ADDRESS_LENGTH = 32;
+    uint256 public constant COSMOS_ADDRESS_LENGTH = 40;
+    uint256 public constant POLKADOT_ADDRESS_LENGTH = 44;
+    uint256[] public lenghts = [
+        EVM_ADDRESS_LENGTH,
+        SOLANA_ADDRESS_LENGTH,
+        COSMOS_ADDRESS_LENGTH,
+        POLKADOT_ADDRESS_LENGTH
+    ];
+
+    function setUp() public override {
+        vm.startPrank(OWNER);
+        super.setUp();
+        initTokenBridge(ARBITRUM_AXELAR_GATEWAY, ARBITRUM_AXELAR_GAS_SERVICE);
+        tokenAddress = address(token);
+        vm.stopPrank();
+    }
+
+    /// @notice it tests the batchDisableAddressLength function can only be called by the owner
+    function test_batchDisableAddressLength_shouldRevert_ifCallerIsNotOwner()
```

Comment:
```suggestion
    function test_batchDisableAddressLength_revertsWhen_callerIsNotOwner()
```
And everywhere else!

Reply:
fixed here (f2fa56f60b759725ca67691a8c822e4a403a2001)